fix(Alerts): Replaced "violation" with "incident" (#14123)
newrelic/docs-website: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

      </td>

      <td>

>         Correlation is activated because the New Relic NRQL alert thresholds and NRQL query are the same. Relevant events from the same [NRQL alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) will be identified. This decision helps relate issues that have the same transaction query latency deviation for example. 

Reviewer: ```suggestion
        Correlation is activated when the entity name with an exceeded threshold and NRQL query are the same. Relevant events from the same [NRQL alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) will be identified. This decision helps relate issues that have the same transaction query latency deviation for example. 
```
